metomi/rose/config_editor/valuewidget/source.py

```diff
@@ -91,7 +91,7 @@ def __init__(self, value, metadata, set_value, hook, arg_str=None):
         frame.add(self._listview)
         value_vbox = Gtk.Box(orientation=Gtk.Orientation.VERTICAL)
         value_vbox.show()
-        value_vbox.pack_start(frame, expand=False, fill=False, padding=0)
+        value_vbox.pack_start(frame, expand=True, fill=True, padding=0)
```

Comment:
Deliberate change to get the "Included" bits to display. Without it, unless you click away from rose edit, it won't show it.